src-tauri/src/file.rs

```diff
@@ -1,15 +1,21 @@
 use std::io::{Write, Read};
 use std::fs::File;
 
+const BUFFER_SIZE: usize = 4096;
```

Comment:
None of this works. These variables are not available anymore in the scope of the `compress()` and `decompress()` functions. Rust compiler also complains about this.